src/olympia/activity/tests/test_utils.py

```diff
@@ -516,7 +516,7 @@ def test_send_activity_mail():
     assert len(mail.outbox) == 1
     assert mail.outbox[0].body == message
     assert mail.outbox[0].subject == subject
-    reference_header = '{addon}/{version}@{site}'.format(
+    reference_header = '<{addon}/{version}@{site}>'.format(
```

Comment:
looking at many other `References` and `In-Reply-To` headers this is perfectly valid, there's other resources like https://cr.yp.to/immhf/thread.html that mention it. I fail to find the relevant place in the RFC though, email RFCs are bad.